src/CoreBundle/Bridge/Chameleon/Module/Sidebar/TableMenuItemProvider.php

```diff
@@ -27,7 +27,7 @@ public function createMenuItem(\TdbCmsMenuItem $menuItem): ?MenuItem
         return new MenuItem(
             $menuItem->fieldName,
             $menuItem->fieldIconFontCssClass,
-            PATH_CMS_CONTROLLER."?pagedef=tablemanager&id={$tableConf->id}"
+            PATH_CMS_CONTROLLER."?pagedef=tablemanager&id={$tableConf->id}&_histid=0&_rmhist=true"
```

Comment:
Do we also need adjustments in `ModuleMenuItemProvider`?

Reply:
No... or maybe... we need to see how the user experience with the new behavior is. At the moment the breadcrumb is not overwritten intentionally if i click on a module like the media manager in the sidebar. This allows me to switch to the media upload without losing my click path and i'm able to go back to the last opened list and editors. I'm not 100% sure if this is a good idea, because it is not really a breadcrumb then, but it never was... it is a click path. We need to test and decide if we should reset the breadcrumb here.